lsdcServer

```diff
@@ -1,3 +1,3 @@
-dzdo pkill -KILL -f daq_main2.py
+pkill -KILL -f daq_main2.py
```

Comment:
Have you considered using a pidfile? Relying on `[p]kill` in this way is unusual.

Reply:
This was a holdover from manually killing all other instances of the LSDC server - they could have been started by multiple local contacts.
Thinking about this a bit more, would I be able to count on systemd killing all processes spawned by launching LSDC server? if so, then I don't even need the line at all.
I don't know enough about how the main process and related processes are handled by systemd

Reply:
OK, we can let this through for now but it should be revisited.

Reply:
actually, this issue seems to be resolved with my Ansible pull request, where using Type=exec and having Conflict settings on the unit ensure that the "other" of lsdc-server and lsdc-server-dev that is not currently running will be stopped, if it had been running.

if this is satisfactory, please resolve this conversation